src/store/poolSlice.ts

```diff
@@ -61,6 +63,22 @@ export type PoolReserve = {
   userReserves?: UserReserveDataHumanized[];
 };
 
+//TODO: temp, move to DebtSwitchActions once types are finalized
+interface DebtSwitchActionsProps {
```

Comment:
Looks like there's a lot of overlap between this interface and the one that's already in `DebtSwitchActions`. Probably just need to consolidate the two in that component.

Reply:
Consolidated to DebtSwitchActionProps